### ISSUE TITLE
Split Terminals - Maintain multiple default shells

### ISSUE THREAD

**Issue by ktmeagher** [microsoft/vscode#90529]
I have been developing in Node and Angular. I typically use PowerShell for running Node and CMD for running Angular. Both from a split-terminal window so I can watch the consoles as I run code. VS Code only allows for a single "Default Shell" for using Terminal. Therefore, I am constantly switching my default shell in order to open both Powershell and CMD. 

I request that this configuration/preference be saved in VS Code so that I don't need to go through the process of reassigning the default terminal in order to open both shells in split view. 

I can imagine other workloads where even more terminal types are used and that could become cumbersome having to constantly change the default shell as you open new terminals into the split view every time you open a project to work on.

A "Save my Terminal" option would allow us to set a custom named configuration to select from the dropdown. So, instead of choosing "Select Default Shell," we could choose "My Custom Terminal" from the dropdown and it would open a saved configuration using multiple shells in the terminal window. This would improve efficiency of getting started. This is especially helpful when opening multiple projects being worked on in parallel in multiple VS Code instances.

Thank you for your consideration of this feature request.

**Comment by ktmeagher** [comment]
> (Experimental duplicate detection)
> Thanks for submitting this issue. Please also check if it is already covered by an existing one, like:
> 
> * [Splitting terminal opens default shell instead of current one (#73194)](https://www.github.com/microsoft/vscode/issues/73194)

My feature request is not duplicating the referred issue. I did not see any request similar to mine while searching. Thank you.

**Comment by Tyriar** [comment]
We defer that sort of behavior to extensions which can enable a greater amount of customization than we could offer built-in, for example https://marketplace.visualstudio.com/items?itemName=fabiospampinato.vscode-terminals

Also related: https://github.com/microsoft/vscode/issues/45444
